src/components/common/ColonyCreationWizard/StepTokenChoice.tsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { useMediaQuery } from 'react-responsive';
+
+import { WizardStepProps } from '~shared/Wizard';
+import Heading from '~shared/Heading';
+import ExternalLink from '~shared/ExternalLink';
+import DecisionHub from '~shared/DecisionHub';
+import { Form } from '~shared/Fields';
+import { multiLineTextEllipsis } from '~utils/strings';
+import { SELECT_NATIVE_TOKEN_INFO } from '~constants';
+
+import { FormValues, Step2 } from './ColonyCreationWizard';
+
+import { query700 as query } from '~styles/queries';
+import styles from './StepTokenChoice.css';
+
+const LEARN_MORE_URL = SELECT_NATIVE_TOKEN_INFO;
+
+const displayName = 'common.ColonyCreationWizard.StepTokenChoice';
+
+const MSG = defineMessages({
+  heading: {
+    id: `${displayName}.heading`,
+    defaultMessage: 'Choose a native token for {colony}',
+  },
+  subtitle: {
+    id: `${displayName}.subtitle`,
+    defaultMessage: `You earn reputation in a colony when it pays you in its\
+      native token.`,
+  },
+  subtitleWithExample: {
+    id: `${displayName}.subtitleWithExample`,
+    defaultMessage: `Leia completes a task for 5 FOX in the ShapeShift colony.
+    Because FOX is the native token of the ShapeShift colony,
+    she also earns 5 reputation in that colony.`,
+  },
+  button: {
+    id: `${displayName}.button`,
+    defaultMessage: 'Back',
+  },
+  notSure: {
+    id: `${displayName}.notSure`,
+    defaultMessage: 'Not sure?',
+  },
+  createTokenTitle: {
+    id: `${displayName}.newToken`,
+    defaultMessage: 'Create a new token',
+  },
+  selectTokenTitle: {
+    id: `${displayName}.existingToken`,
+    defaultMessage: 'Use an existing ERC20 token',
+  },
+  createTokenSubtitle: {
+    id: `${displayName}.newTokenSubtitle`,
+    defaultMessage: 'For example: MyAwesomeToken',
+  },
+  selectTokenSubtitle: {
+    id: `${displayName}.existingTokenSubtitle`,
+    defaultMessage: 'Add in the list of examples: UNI, SUSHI, & AAVE',
+  },
+  tooltipCreate: {
+    id: `${displayName}.tooltipCreate`,
+    defaultMessage: `Good for projects that don't already have a token or who want more control over their token`,
+  },
+  tooltipSelect: {
+    id: `${displayName}.tooltipSelect`,
+    defaultMessage: `Good for projects that already have their own token or want to use an existing one like DAI.`,
+  },
+});
+
+const options = [
+  {
+    value: 'create',
+    title: MSG.createTokenTitle,
+    subtitle: MSG.createTokenSubtitle,
+    icon: 'question-mark',
```

Comment:
We really need to refactor `Icon`s to use `enum`s rather then string names.

Can you please open a maintenance  issue for that?

Reply:
Sure, https://github.com/JoinColony/colonyCDapp/issues/92